<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Stammdaten</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelName">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditName">
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelPLZ">
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>PLZ</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditPLZ">
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelGender">
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Geschlecht</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxGender">
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <item>
                 <property name="text">
                  <string>&lt;geschlecht auswählen&gt;</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>männlich</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>weiblich</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>divers</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Präferenzen</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Verfügbare Aktienmärkte</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="listWidgetAvailibleSharesList">
                  <property name="maximumSize">
                   <size>
                    <width>500</width>
                    <height>1000</height>
                   </size>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::ExtendedSelection</enum>
                  </property>
                  <property name="sortingEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::MinimumExpanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>50</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonAddTo">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonClearAll">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>DEL</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonRemoveFrom">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>&lt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::RightToLeft</enum>
                  </property>
                  <property name="text">
                   <string>Meine Präferenzen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="listWidgetUserSharesList">
                  <property name="maximumSize">
                   <size>
                    <width>500</width>
                    <height>1000</height>
                   </size>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::ExtendedSelection</enum>
                  </property>
                  <property name="sortingEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Bezahlung</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QFormLayout" name="formLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <property name="rowWrapPolicy">
               <enum>QFormLayout::DontWrapRows</enum>
              </property>
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="horizontalSpacing">
               <number>33</number>
              </property>
              <property name="verticalSpacing">
               <number>23</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="labelIBAN">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>IBAN</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLineEdit" name="lineEditIBAN">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelBankName">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Bankname</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QGroupBox" name="groupBoxSubscriptions">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="title">
                 <string>Abrechnungs Zeitraum</string>
                </property>
                <widget class="QRadioButton" name="radioButtonMonthly">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>20</y>
                   <width>121</width>
                   <height>22</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>monatlich</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="radioButtonYearly">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>40</y>
                   <width>121</width>
                   <height>22</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>jährlich</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>50</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0" colspan="2">
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelBankNameOutput">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <bold>false</bold>
                  <underline>false</underline>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Unbekannte Bank</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelPPM">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Preis pro Monat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPPMOutput">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxSaveData">
            <property name="text">
             <string>Daten speichern</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonQuit">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Beenden</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOrder">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Bestellen</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
